test/check-machines-multi-host-consoles

```diff
@@ -49,6 +49,10 @@ class TestMultiMachineVNC(VirtualMachinesCase):
         m2 = self.machine2
         m2_host = "10.111.113.2"
 
+        user = None
+        if "suse" in m.image:
+            user = "root"
```

Comment:
This doesn't look c-machines specific. I figure this should rather go into https://github.com/cockpit-project/cockpit/blob/main/test/common/testlib.py#L931 ?

Reply:
I'm happy to move that there. The API allows setting the username so this felt like the right place to do it.

Reply:
Yes, but I mean "sudo asks for root password" is a general property of the opensuse image, so cockpit etc. tests are also going to need this anyway, no?

Reply:
You're correct. Created a PR for the change: https://github.com/cockpit-project/cockpit/pull/20525